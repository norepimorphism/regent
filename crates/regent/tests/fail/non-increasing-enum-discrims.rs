fn main() {
    #[regent::bitwise]
    enum A { _0 = 1, _1 = 0 }
}
