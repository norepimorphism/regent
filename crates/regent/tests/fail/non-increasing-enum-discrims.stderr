error: variant discriminant must be greater than the previous
 --> tests/fail/non-increasing-enum-discrims.rs:3:27
  |
3 |     enum A { _0 = 1, _1 = 0 }
  |                           ^
